src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -373,10 +357,7 @@ where
                                     }
                                     untimestamped_messages.entry(pid).or_default().push((message, offset));
                                 }
-                                SourceMessageType::SourceStatus(update) => {
-                                    if let Some(healthchecker) = &mut healthchecker {
-                                        healthchecker.update_status(update).await;
-                                    }
+                                SourceMessageType::SourceStatus(_update) => {
```

Comment:
what's the plan for this one?

Reply:
I expect we'll want this, or some evolution of it, to get transient/indefinite errors from the source to the healthchecker... right now they're retried indefinitely, so this code can't learn about them otherwise. I'll get this used and/or cleaned up in a future PR.